src/main/java/com/couchbase/client/dcp/message/DcpDeletionMessage.java

```diff
@@ -58,12 +58,17 @@ public static long revisionSeqno(final ByteBuf buffer) {
         return buffer.getLong(MessageUtil.HEADER_SIZE + 8);
     }
 
+    public static short nmeta(final ByteBuf buffer) {
+        return buffer.getShort(MessageUtil.HEADER_SIZE + 16);
```

Comment:
is this one always there? Otherwise we may should add some checks to see if it is?